plugins/modules/katello_repository.py

```diff
@@ -56,7 +56,7 @@
   url:
     description:
       - Repository URL to sync from
-    required: true
+    required: false (in case you're modifying an existing repository)
```

Comment:
```suggestion
    required: false
```
`required` is a special param for an Ansible documentation block, so passing a string here might not have the expected result. Also, I don't think it's actually required when you create a new repo, as it's totally fine to have a repository that has no "upstream" to sync from.

Reply:
this actually failed on travis:

```
ansible.errors.AnsibleError: Invalid required value 'false (in case you're modifying an existing repository)' for parameter 'url' in 'katello_repository' (must be truthy)
```